modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -6,17 +6,17 @@
 [id="virt-creating-bridge-nad-cli_{context}"]
 = Creating a Linux bridge network attachment definition in the CLI
```

Comment:
NetworkAttachmentDefinition ?

Reply:
According to our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting), if you are referring to the actual instance of the API object, use NetworkAttachmentDefinition . If it is a general or conceptual reference, use "network attachment definition".

In this case, it is a general reference, so we use "network attachment definition".

Reply:
But isn't this a reference to the API object? The header talks about creating an instance of a NetworkAttachmentDefinition, isn't it?

Reply:
Hi @yossisegev, yes, it's a reference to the API object, but it's not referring to a _specific_ instance of that object. 

This is an example of a specific API object reference, which requires the PascalCase and backticks:
```
. Create a `NetworkAttachmentDefinition` manifest.
```

In contrast, the contribution guide provides this example of a general reference to API objects: "You must have at least one secret, config map, or service account."

For headings like this one, we avoid using specific object references (like "Creating a `NetworkAttachmentDefinition` object") because of the following requirement from the contrib guide: "Do not use backticks or other markup in assembly or module headings."

Reply:
Thanks for this clarifying explanation, Pan.
I just hope I can remember that, for future doc reviews :)